cmd/cl-adm/templates/k8s.go

```diff
@@ -91,7 +91,7 @@ spec:
         - name: cl-controlplane
           image: cl-controlplane
           imagePullPolicy: IfNotPresent
-          args: ["--log-level", "info"]
+          args: ["--log-level", "info", "--deployment", "k8s"]
```

Comment:
deployment could be changed to runtimeEnv or platform as suggested by @orozery

Reply:
Done